capture-sdk/sdk/src/main/java/net/gini/android/capture/EntryPoint.kt

```diff
@@ -0,0 +1,22 @@
+package net.gini.android.capture
+
+/**
+ * Created by Alpár Szotyori on 10.07.23.
+ *
+ * Copyright (c) 2023 Gini GmbH.
+ */
+
+/**
+ * Use this enum to set which entry point is used for starting the SDK.
+ */
+enum class EntryPoint {
```

Comment:
[Clarification]: What do you think about putting the **BUTTON** at the 1st enum position?
It's going to be a default value and alphabetically is higher.